corehq/apps/app_manager/migrations/0001_initial.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+from django.conf import settings
+from django.core.management import call_command
+
+from dimagi.utils.couch import sync_docs
+
+import corehq.apps.app_manager.models.common as models_common
+import corehq.apps.app_manager.models.schedules as models_schedules
+
+def sync_app_manager_docs(apps, schema_editor):
+    sync_docs.sync(models_common, verbosity=2)
+    sync_docs.sync(models_schedules, verbosity=2)
+
+
+class Migration(migrations.Migration):
+
+    dependencies = []
+
+    operations = [
+        migrations.RunPython(sync_app_manager_docs),
+    ]
```

Comment:
Can you explain what this is doing? It looks like the intention of this code is to make app_manager design docs get picked up by couchdbkit's code and synced like the other apps', and that that wasn't happening because couchdbkit looks for a models.py with a Document subclass in it (or something along those lines). Assuming I'm on track so far, a few things:
- I don't think this is an effective solution because the migration will get run once, but we want this to get run every time we syncdb (which I guess is now just called `migrate`).
- I don't see why we'd want to sync `models_common` and `models_schedules` separately, since the only thing getting sync is design docs, and those live outside of the whole `models` situation—what we really are trying to sync is the "app", `app_manager`
- If we want this to be a pattern, I think we certainly don't want to have to drop a file like this in each app where we're splitting models

Given these, it seems to me the best solution is to find where the logic is that decides whether to skip over an app, and change it to include apps with a `models` directory